src/components/ebay-menu/index.marko

```diff
@@ -50,6 +51,9 @@ $ var baseClass = input.classPrefix || "menu";
                 var checked = component.isChecked(index);
             }
             <if (item.separator)>
+                $ {
+                    separatorCount += 1;
+                }
```

Comment:
I think you can just do `$ separatorCount += 1;` in 1 line.

Reply:
Cool. Will do that.

Reply:
Forgot to push this change. Let me do this.